namespaces/Representor.md

```diff
@@ -0,0 +1,44 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Element)
+
+The Resource represents a Hypermedia Resource, with its available transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `attributes` (object)
+    - `transitions` - (array[Transition]) - The available transitions from this resource.
+    - `resources` - (array[Resource]) - Any embedded (resources) from this resource.
+    - `relation` - (string) - An optional relation to this resource, applicable when embedded in another resource.
+- `content` (object) - Any properties of the resource.
+
+## Transition (Object Element)
+
+A transition is an available progression from one state to another state.
+
+### Properties
+
+- `element`: transition (string, fixed)
+- `attributes` (object, required)
+    - `relation` - (string, required) - The relation of this transition from the current resource.
+    - `parameters` ([Object Type][]) - A collection of parameters for this transition.
+    - `attributes` ([Object Type][]) - A collection of possible attributes for this transition.
```

Comment:
> Thinking about this even more – if you do not want elements in plain attributes hash – maybe the href should be attribute and content of the transition SHOULD be input properties – parameters and attributes elements.

it's not that it's what I want (I'm no dictator here!), it just feels like it's not designed correctly. I'm not against this practice of using elements like this, but I think it actually complicates things. The attributes hash is a shortcut for a fully-refracted object (array of member elements). You can do this refracting infinitely as a refracted member element itself has attributes. What happens when you refract attributes that contain already-refracted elements?

How you have described it here is how I have it in my mind. Links (IMO) look very similar to HTML `<link>` or `<a>` tags. Forms are the same. The benefit we have is that we can include MSON into a "form" and provide a JSON-like input rather than key/value inputs like with HTML `<inputs>`.

Just to circle back around. Refract is suppose to free you from complex JSON structures to use simple, single-purpose elements. Overloading attributes is going back to the old ways (which again, is not evil :) ).

> You MAY need a description of data structure here.

Could you use the description meta property for this description?